src/app/components/Item/ItemTableRow.jsx

```diff
@@ -64,7 +81,7 @@ class ItemTableRow extends React.Component {
     const status = item.status && item.status.prefLabel ? item.status.prefLabel : ' ';
     let itemRequestBtn = status;
 
-    if (item.aeonUrl) {
+    if (item.aeonUrl && features.includes('aeonLinks')) {
```

Comment:
Minor thing, but the precedent has been shish-kabab case for feature names (e.g. "on-site-edd", "drb-integration", "my-account"), so might be helpful long-term to standardize on that

Reply:
Fixed